service/lib/agama/storage/finisher.rb

```diff
@@ -142,9 +145,12 @@ def run?
         end
 
         def run
-          target = File.join(Yast::Installation.destdir, UDEV_RULES_DIR)
-          FileUtils.mkdir_p(target)
-          FileUtils.cp(glob_files, target)
+          glob_files.each do |file|
+            target = File.dirname(file).sub(root_dir, dest_dir)
```

Comment:
This relies in the fact that `dest_dir` includes a final slash. That is, is "/mnt/". It would not work if `dest_dir` is "/mnt" (without the trailing slash). But I guess we can live with that.